cmd/migrate/postgres.go

```
@@ -17,21 +18,89 @@ import (
+		fmt.Print(string(raw))
+		fmt.Println("Migration process completed")
+		return nil
+	}
+
+	l.Info(ctx, "Applying PostgreSQL SQL file", "host", cfg.Postgres.Host, "file", file)
+
+	db, err := sql.Open("postgres", cfg.Postgres.GetDSN())
+	if err != nil {
+		return fmt.Errorf("failed to connect to postgres: %w", err)
+	}
+	defer db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)